server/server/models/user.py

```diff
@@ -69,3 +95,30 @@ def is_active(self, request):
                                 .filter_by(status="Active").one().id
                 return True
         return False
+
+    def __init__(self, email, nickname, password, create_date, location, first_name, last_name, status_id, role_id, avatar, banned_to_date):
+        self.email = email
+        self.nickname = nickname
+        self.password = password
+        self.create_date = create_date
+        self.location = location
+        self.first_name = first_name
+        self.last_name = last_name
+        self.status_id = status_id
+        self.role_id = role_id
+        self.avatar = avatar
+        self.banned_to_date = banned_to_date
+
+def users_to_json(obj):
```

Comment:
It's a good candidate to be converted to method of `User` model.

Also according to https://stackoverflow.com/questions/1958219/convert-sqlalchemy-row-object-to-python-dict you may use something like `inspect(obj).mapper.column_attrs` or `obj.__table__.columns` (depending of SQLAlchemy version) to get column names instead of hardcoding.